src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
+
+  &__body {
```

Comment:
The `&__body` class should be applied to HTML `body` tag, but according to BEM, it's better not to style the `body` tag directly. Consider changing the class name to something like `&__container` and apply it to a `div` wrapper inside the `body` tag. 🚀